### PR TITLE
Side w1

### PR DESCRIPTION
**Pull request by luciano-ilha**
This project contains a clone of the Newsweek website, was done with HTML, CSS Positioning and Layout (Including Grid and Flexbox), Media Query, and Bootstrap, using mobile-first approach. After finish mobile-first, done 768px view and 992px for better responsiveness.

These are the main concepts applied in this project:

CSS Positioning and Layout (Including Grid and Flexbox)
Media Query and Overall Responsiveness
Image embedding
Hover effects and other dynamic responses

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes:recycle: :traffic_light: 

Hi @luciano-ilha, great job with this project :+1: Professional commit messages :100: Very descriptive PR :slightly_smiling_face: README professional :100: 
Let us improve our code and make it better by addressing the comment below:
- Files not related to the project must not be committed, let add it to a gitignore file
gih
- The Newsletter and the Subscribe must have the same height. Currently
ffr

Kindly make these changes and submit another code review :slightly_smiling_face: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by peterrobert** [comment]
### PULL REQUEST UNDER REVIEW BY PETER ROBERT (TSE)

File: styles/style.css [hunk 1]
```diff
@@ -0,0 +1,1182 @@
...
+  content: "";
+  display: block;
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 4.125rem;
+  background-color: #f72210;
+  border-left: 0.75rem solid white;
+  border-top: 0.8125rem solid #f72210;
+}
```
Comment (peterrobert) [inline styles/style.css]: Try using bootstrap display block class [here](https://getbootstrap.com/docs/4.0/utilities/display/) is a link to bootstrap display property

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,1182 @@
...
+
+.header-bottom-list {
+  display: flex;
+  justify-content: center;
+  color: #000;
+  width: 100%;
+  font-size: 1.1875rem;
+  line-height: 1.5rem;
+  font-weight: bold;
+}
```
Comment (peterrobert) [inline styles/style.css]: Try using bootstrap display block class [here](https://getbootstrap.com/docs/4.0/utilities/display/) is a link to bootstrap display property

**Review by peterrobert — COMMENTED** [review]
### CHANGES REQUIRED :recycle: 
A great project so far :+1: Everything is working perfectly. Great commit messages :+1:  However there is a few things that require to be changed before i can approve this project. 

- [x] Kindly check the links to some of your images. They appear broken.

broken_link_ing

- [x] I have added other changes below :arrow_down: 

**When you through kindly submit the project for another review**

**If you have any questions kindly reach out to me via slack** @peterrobert 

### HAPPY CODING

File: styles/style.css (hunk 1 above)
Comment (luciano-ilha) [reply styles/style.css]: Done! Thanks for the help

File: styles/style.css (hunk 2 above)
Comment (luciano-ilha) [reply styles/style.css]: d-flex you mean?! Done! Tks for the help

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @luciano-ilha     :handshake: 

### Good job so far :clap:  your project looks like the original :medal_sports: 

### there are some changes to get the project approved :point_down:
- You should always check your project using the validators before asking for a review, please fix all the errors:
Screenshot from 2020-06-08 13-59-05
- [x] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. (Students were asked to use this one, so in most cases their code should be already validated) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)
- Please do as the previous TSE asks and remove files not related to the project, (I am talking about .DS_store):
- [x] Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- Please remove the comment from you code:
- [x] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
screenshot

### Once your done with the changes, ask for another code review.

**Comment by voscarmv** [comment]
# Pull Request Under Review by Oscar Mier (TSE) 👨‍💻 🐦

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work so far @luciano-ilha 👀 👏 👏 👏 
## However, a few changes are still required 🧰 🔨 🔧 
- Your use of the semantic `<nav>` and `<footer>` tags is good, but I don't see any `<main>` tag indicating the central content of your page. I also don't see any `<section>` tags being used to indicate the different sections in your page. Please improve your use of Semantic HTML and re-submit for review! Screenshot:
image

### Once you implement these changes re-submit for review 😄 👍 
## Questions? Comments? Contact me on [Twitter](http://www.twitter.com/voscarm) or [Linkedin](http://www.linkedin/in/oscar-mier) 🧑‍💻 💬 🧑‍💻 
## Happy hacking! 🧠 ⌨️ 🖥️

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello @luciano-ilha** 
Amazing job :clap: :clap: you have passed all the requirements for this project
Now you are ready to continue with your next challenge :muscle: 
Don't forget to merge with the master branch

Keep going like this,

Oscar Andrés Russi Porras

**Pull request merged**